### PR TITLE
Fix typo on README and improve the script execution

### PR DESCRIPTION
**Pull request by leoheck**
Just a fix for a typo in the command line.

### CODE REVIEW

**Comment by leoheck** [comment]
1 Fix a typo on README.md of FritzFrog script.
2. It also adds the interpreter of the script in the first line (it was not working by default on my zsh, for instance)
3. Enables the execution flag of the script, so we can run it as stated in the README.md

This was the incompatibility on item number 2
```
./detect_fritzfrog.sh: 4: Syntax error: "(" unexpected
```

File: FritzFrog/detect_fritzfrog.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
+
```
Comment (d97jro) [inline FritzFrog/detect_fritzfrog.sh:2]: The script is more portable if you use

`#!/usr/bin/env bash`

Comment (leoheck) [reply FritzFrog/detect_fritzfrog.sh:2]: Got it. Conflict fixed.

**Comment by ophirharpazg** [comment]
Thank you guys 👍 This is great.

**Pull request merged**